conda/recipes/pynvjitlink/conda_build_config.yaml

```
@@ -0,0 +1,8 @@
+c_compiler_version:
+  - 11
+
+cxx_compiler_version:
+  - 11
+
+cuda_compiler:
+  - cuda-nvcc
```

This will ensure we use the latest version of CUDA 12 available

```suggestion
cuda_compiler:
  - cuda-nvcc

cuda_compiler_version:
  - 12
```